test/test_ca_geocoder.rb

```diff
@@ -5,34 +5,34 @@
 class CaGeocoderTest < BaseGeocoderTest #:nodoc: all
   CA_SUCCESS = <<-EOF
   <?xml version="1.0" encoding="UTF-8" ?>
-  <geodata><latt>49.243086</latt><longt>-123.153684</longt></geodata>
+  <geodata> <latt>49.243044</latt> <longt>-123.154975</longt> <postal>V6L2J7</postal> <standard> <stnumber>2105</stnumber><staddress>32nd AVE W</staddress><city>Vancouver</city><prov>BC</prov><confidence>0.8</confidence></standard> </geodata>
   EOF
 
   def setup
-    @ca_full_hash = {street_address: "2105 West 32nd Avenue", city: "Vancouver", state: "BC"}
+    @ca_full_hash = {street_address: "2105 West 32nd Avenue", city: "Vancouver", province: "BC", state: "BC"}
```

Comment:
Line is too long. [109/80]<br>Space inside { missing.<br>Space inside } missing.